docs/rfcs/2023-07-06-table-engine-refactor.md

```diff
@@ -95,8 +95,8 @@ subgraph Frontend["Frontend"]
     end
 end
 
-MyTable --> MetaSrv
-MetaSrv --> ETCD
+MyTable --> Metasrv
+Metasrv --> ETCD
```

Comment:
Will there be any issue if I make this change? cc @evenyag

Reply:
This should be fine